usr/src/uts/i86pc/io/vmm/vmm.c

```diff
@@ -645,6 +650,12 @@ vm_cleanup(struct vm *vm, bool destroy)
 
 		VMSPACE_FREE(vm->vmspace);
 		vm->vmspace = NULL;
+
+#ifndef __FreeBSD__
+		arc_virt_machine_release(vm->arc_resv >> PAGE_SHIFT);
```

Comment:
After what I said in vm_arc_resv()... should we store #-of-pages instead of the actual length in bytes?